workflows/ampliseq.nf

```diff
@@ -604,7 +617,7 @@ workflow AMPLISEQ {
     if ( params.sbdiexport ) {
         SBDIEXPORT ( ch_dada2_asv, ch_dada2_tax, ch_metadata )
         ch_versions = ch_versions.mix(SBDIEXPORT.out.versions.first())
-        SBDIEXPORTREANNOTATE ( ch_dada2_tax )
+	    SBDIEXPORTREANNOTATE ( ch_dada2_tax, ch_barrnapsummary )
```

Comment:
```suggestion
        SBDIEXPORTREANNOTATE ( ch_dada2_tax, ch_barrnapsummary )
```
Random tab

Reply:
Good find! Also fixed in https://github.com/nf-core/ampliseq/pull/550